packages/components/addon/components/hds/stepper/stepper-indicator/index.hbs

```diff
@@ -0,0 +1,22 @@
+<div class={{this.classNames}} ...attributes>
+  <div class="svg-container">
+    <svg width="20" height="23" viewBox="0 0 20 23" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path 
+        d="M1.6726 5.76389L8.65253 1.34541C9.46819 0.829074 10.5082 0.82849 11.3245 1.34391L18.3247 5.76421C19.05 6.22224 19.4899 7.02019 19.4899 7.87806V15.1219C19.4899 15.9798 19.0501 16.7777 18.3247 17.2358L11.3245 21.6561C10.5083 22.1715 9.46819 22.1709 8.65253 21.6546L1.6726 17.2361C0.948601 16.7778 0.509766 15.9806 0.509766 15.1238V7.87623C0.509766 7.01936 0.948602 6.22219 1.6726 5.76389Z" 
+        fill="#F2F8FF"
+        stroke="#0C56E9"
+      />
+    </svg>
+  </div>
+  <div class="indicator-status">
+    {{#if @isProcessing}}
+        <FlightIcon @name="loading" @size="16" />
+      {{else if (eq @status "complete")}}
+        <FlightIcon @name="check" @size="16" />
+      {{else if (eq @status "incomplete")}}
+        <span>{{@stepNumber}}</span>
+      {{else if (eq @status "inProgress")}}
+        <span>{{@stepNumber}}</span>
+    {{/if}}
```

Comment:
question: is `status` only checked if `isProcessing` is true? If so, this section needs a little bit of refactoring; use a nested `{{#if}}` here, and put the default state into the `{{else}}` at the end before the closing `{{/if}}`. Here's some documentation: https://guides.emberjs.com/release/components/conditional-content/#toc_usage